SideMenuTransitionTests/SideMenuDismissTransitionTests.swift

```diff
@@ -28,20 +22,20 @@ final class SideMenuDismissTransitionTests: XCTestCase {
   }
 
   func test_transitionDuration() {
-    XCTAssertEqual(sut.transitionDuration(using: nil), 0.25)
+    expect(self.sut.transitionDuration(using: nil)) == 0.25
   }
 
   func test_animateTransition() {
     let context = MockViewControllerContextTransitioning()
 
     sut.animateTransition(using: context)
 
-    XCTAssertTrue(sut.viewAnimator is MockUIViewAnimator.Type)
-    XCTAssertEqual(menuAnimator.didAnimateToProgress, 0)
-    XCTAssertEqual(menuAnimator.didAnimateInContainerView, context.containerView)
+    expect(self.sut.viewAnimator).to(beAKindOf(MockUIViewAnimator.Type.self))
```

Comment:
@darrarski Could you take a look at this? I think it looks too... extensive. idk...

Reply:
🤔 If this PR’s goal is to improve readability of the assertions, I would say it’s better to stay with

```swift
XCTAssertTrue(shut.viewAnimator is MockUIViewAnimator.Type)
```

Of course you can use Nimble’s `expect` function in other places, whenever it makes the code more readable 😄

Reply:
TBH, I don’t have a strong opinion in this case. Using `expect` may help in case of the assertion failure (better failure message). I will leave it for you to decide which one to use here.

Reply:
I'm tempted to use nimble `description` inline parameter for this case but honestly w/ XCTAssert this case already looks fine, besides, this would get even more stretched.  I'll not use `XCTAssertTrue` but instead: `XCTAssertEqual(sut.viewAnimator is MockUIViewAnimator.Type, true)`. I think it looks more readable.